CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
-## [0.6.0](https://github.com/eparreno/rack-jwt/tree/v0.5.0) (2020-09-29)
+## [0.7.0](https://github.com/eparreno/rack-jwt/tree/v0.7.0) (2021-04-27)
```

Comment:
[nit] Don't forget to update the date here

Reply:
👍 Date updated